modular_skyrat/master_files/code/modules/admin/admin.dm

```diff
@@ -0,0 +1,20 @@
+GLOBAL_VAR_INIT(dchat_allowed, TRUE)
+
+/datum/admins/proc/toggledchat()
+	set category = "Server"
+	set desc="Toggle dis bitch"
+	set name="Toggle Dead Chat"
+	toggle_dchat()
+	log_admin("[key_name(usr)] toggled OOC.")
+	message_admins("[key_name_admin(usr)] toggled OOC.")
+	SSblackbox.record_feedback("nested tally", "admin_toggle", 1, list("Toggle OOC", "[GLOB.ooc_allowed ? "Enabled" : "Disabled"]")) //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
```

Comment:
Shouldn't it say deadchat instead of OOC?

Reply:
refresh the page